Detectors/TRD/base/test/testTRDDiffusionCoefficient.cxx

```diff
@@ -36,55 +36,16 @@ BOOST_AUTO_TEST_CASE(TRDDiffusionCoefficient_test1)
   TGeoGlobalMagField::Instance()->Lock();
 
   auto commonParam = CommonParam::instance();
+  DiffusionAndTimeStructEstimator estimator;
   float dl = 0;
   float dt = 0;
   float vd = 1.48;
-  commonParam->getDiffCoeff(dl, dt, vd);
+  estimator.getDiffCoeff(dl, dt, vd);
   // check whether the values match the expected AliRoot known output
   BOOST_CHECK_CLOSE(dl, 0.0255211, 0.1);
   BOOST_CHECK_CLOSE(dt, 0.0179734, 0.1);
 }
 
-/// \brief Test time structure
-BOOST_AUTO_TEST_CASE(TRDTimeStructure_test)
-{
-  auto commonParam = CommonParam::instance();
-  DiffusionAndTimeStructEstimator estimator;
-  BOOST_CHECK_CLOSE(estimator.TimeStruct(1.48, 1., 0.1), commonParam->timeStruct(1.48, 1., 0.1), 0.001);
-  BOOST_CHECK_CLOSE(estimator.TimeStruct(1.1, 1., 0.1), commonParam->timeStruct(1.1, 1., 0.1), 0.001);
-  BOOST_CHECK_CLOSE(estimator.TimeStruct(2, 1., 0.1), commonParam->timeStruct(2, 1., 0.1), 0.001);
-  BOOST_CHECK_CLOSE(estimator.TimeStruct(4, 1., 0.1), commonParam->timeStruct(4, 1., 0.1), 0.001);
-}
```

Comment:
Back then, I liked having the two implementations set. However, since the second implementation evolved differently, changes can cause side effects, and these lines are like having a regression test of its functionality. I think it's ok to drop this test and the implementation from commonParam if one remembers that all of this comes from the same code in AliRoot.